Mathlib/RingTheory/Norm/Transitivity.lean

```diff
@@ -106,35 +107,35 @@ lemma eval_zero_det_det : eval 0 (f.polyToMatrix (cornerAddX M k).det).det = (f
   congr; ext; simp [cornerAddX, diagonal, apply_ite]
 
 lemma eval_zero_comp_det :
-    eval 0 (comp m m n n R[X] <| f.polyToMatrix.mapMatrix <| cornerAddX M k).det =
-      (comp m m n n R <| f.mapMatrix M).det := by
+    eval 0 (comp m m n n R[X] <| (cornerAddX M k).map f.polyToMatrix).det =
+      (comp m m n n R <| M.map f).det := by
   simp_rw [← coe_evalRingHom, RingHom.map_det, ← compRingEquiv_apply, ← RingEquiv.coe_toRingHom,
-    ← RingHom.comp_apply, ← RingHom.comp_assoc, evalRingHom_mapMatrix_comp_compRingEquiv,
-    RingHom.comp_assoc, RingHom.mapMatrix_comp, evalRingHom_mapMatrix_comp_polyToMatrix,
-    ← RingHom.mapMatrix_comp, RingHom.comp_apply]
+    ← RingHom.mapMatrix_apply, ← RingHom.comp_apply, ← RingHom.comp_assoc,
+    evalRingHom_mapMatrix_comp_compRingEquiv, RingHom.comp_assoc, RingHom.mapMatrix_comp,
+    evalRingHom_mapMatrix_comp_polyToMatrix, ← RingHom.mapMatrix_comp, RingHom.comp_apply]
   congr with i j
   simp [cornerAddX, diagonal, apply_ite]
 
 theorem comp_det_mul_pow :
-    ((f.mapMatrix M).comp m m n n R).det * (f (M k k)).det ^ (Fintype.card m - 1) =
-      (f (M k k)).det * ((f.mapMatrix mulAuxMatBlock).comp _ _ n n R).det := by
-  trans ((f.mapMatrix (M * auxMat M k)).compRingEquiv m n R).det
-  · simp_rw [_root_.map_mul, det_mul, f.mapMatrix_apply, compRingEquiv_apply,
-      ((auxMat_blockTriangular M k).map f).comp.det_fintype, Fintype.prod_Prop,
+    ((M.map f).comp m m n n R).det * (f (M k k)).det ^ (Fintype.card m - 1) =
+      (f (M k k)).det * (((mulAuxMatBlock).map f).comp _ _ n n R).det := by
```

Comment:
Is there a way to make this work? I tried `notation:max` but it doesn't work ...
```suggestion
      (f (M k k)).det * ((mulAuxMatBlock.map f).comp _ _ n n R).det := by
```

Reply:
I don't know :shrug:
I suggest you ask on Zulip, so that this PR isn't blocked by it.

bors merge